src/redux/books/books.js

```diff
@@ -0,0 +1,40 @@
+const ADD_BOOK = 'library/books/ADD_BOOK';
+const REMOVE_BOOK = 'library/books/REMOVE_BOOK';
+
+const stateInit = [{
+  id: 1, title: 'The Alchemist', category: 'Fiction', author: 'Paulo Coelho',
+},
+{
+  id: 2, title: 'Rich Dad Poor Dad', category: 'Financial', author: ' Robert Kiyosaki and Sharon Lechter',
+},
+{
+  id: 3, title: 'The Way of the Peaceful Warrior', category: 'Biography', author: 'Dan Millman',
+},
+{
+  id: 4, title: 'Think and Grow Rich ', category: 'Financial', author: 'Napoleon Hill',
+}];
```

Comment:
- [ ] According to the project requirement, your `initialState` should return an empty array. Kindly implement it. 👍

Reply:
Thanks for the review changes made 👍